modules/cli/src/main/scala/scala/cli/commands/Shebang.scala

```diff
@@ -0,0 +1,26 @@
+package scala.cli.commands
+
+import caseapp._
+
+final case class ShebangOptions(
```

Comment:
NIT Could `ShebangOptions` be moved to its own file (`ShebangOptions.scala`), like we do for all other classes?

Reply:
Done